### ISSUE TITLE
Nominated Filter -> Toggle visibility of archived

### ISSUE THREAD

**Issue by lmrhody** [PressForward/pressforward#567]
This filter is experiencing some stickiness. When selecting Toggle visibility from the Filter drop down menu, the item switches to "Hide Archived" (that works), but then the breadcrumb trail does not say "hiding/showing archived" (as perhaps it could). Second, once Toggle visibility is selected and the item turns to "hide archived." When "Hide archived" is selected (to return to Toggle visibility of archived), "hide archived" becomes stuck.  The item remains highlighted, and the words remain the same, though the toggling *does* happen. 

In other words, the toggling works fine, it's the filter that doesn't switch between options (wording). 

![screen shot 2015-03-02 at 1 55 01 pm](https://cloud.githubusercontent.com/assets/1395793/6447991/b714deb0-c0e4-11e4-9bf9-2323ea90a6ed.png)
